ui/scripts/inlineHtml/inlineScripts.js

```diff
@@ -1298,13 +1298,16 @@ const updateDefenseValues = function () {
         const defenseIsParry = defense === 'parry';
         const curDefenseName = defenseIsDodge ? dodgeName : defenseIsBlock ? blockName : parryName;
         const cur_defense = getNumberIfValid(values[curDefenseName]);
-        const base = (
+        /**
+         * FIXME - Deprecate `reaction_penalty` since it is redundant with `defense_boost` as a follow up.
+         */
+        const total = (
           cur_defense
           + (defenseIsBlock ? cur_shield : 0)
           + (defenseIsParry ? highestWeaponDefense : 0)
           - Math.abs(getNumberIfValid(values[reactionPenaltyName]))
+          + getNumberIfValid(values[defenseBoostName])
         );
-        const total = base + getNumberIfValid(values['defense_boost']);
```

Comment:
`base` is never used, so combined with above.
Based on this, I think we should remove `reaction_penalty` since it is effectively the same as `defenseBoost`

In doing so, we can then adjust weight placement, and perhaps simplify the header a bit.